.github/workflows/android-ci-pull.yml

```diff
@@ -59,6 +59,9 @@ jobs:
           submodules: recursive
           fetch-depth: 0
 
+      - name: Setup Android SDK
```

Comment:
Why is this needed? The latest SDK should already be set in GitHub actions.

Reply:
https://github.com/maplibre/maplibre-gl-native/actions/runs/3142254689/jobs/5105640527

Reply:
it says the license isn't accepted otherwise